site/src/api/typesGenerated.ts

```diff
@@ -420,6 +421,15 @@ export interface WorkspaceAgentResourceMetadata {
   readonly cpu_mhz: number
 }
 
+// From codersdk/workspaceapps.go:7:6
+export interface WorkspaceApp {
+  readonly id: string
+  readonly name: string
+  readonly command?: string
+  readonly access_url?: string
+  readonly icon: string
```

Comment:
Is icon required? I think we should have some fallback on the frontend and make it optional in terraform if it isn't already.

Reply:
True. I shall fix